src/test/java/com/greenfox/poker/controller/HomeControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.greenfox.poker.controller;
+
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.webAppContextSetup;
+
+import com.greenfox.poker.PokergameApplication;
+import com.greenfox.poker.service.UserService;
+import java.nio.charset.Charset;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import static org.hamcrest.core.Is.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = PokergameApplication.class)
+@WebAppConfiguration
+@EnableWebMvc
+public class HomeControllerTest {
```

Comment:
The name shouldn't be `HomeControllerTest` here. Instead you should call this `UserControllerTest` (after you named your controller to `UserController`), and all the test of this (and just this) controller should be here.